src/app/pages/forms-and-validation/in-a-list/in-a-list.component.scss

```diff
@@ -0,0 +1,38 @@
+@import '~@pxblue/colors/palette.scss';
+
+::ng-deep .pxb-drawer-layout-nav-content {
+    background-color: map-get($pxb-white, 50);
+}
+section.mat-typography {
+    padding-top: 24px;
+    max-width: 686px;
+    margin: 0 auto;
+
+    .list-card {
+        padding: 0;
+        mat-list {
+            padding: 0;
+        }
+    }
+}
+.ip-address {
+    border: 0;
+    background-color: map-get($pxb-white, 200);
+    height: 40px;
+    outline: 0;
+    font-size: 16px;
+    padding: 8px 16px;
+    border-bottom: 1px solid map-get($pxb-gray, 100);
```

Comment:
Can we use the `divider` prop on the InfoListItem here?

Reply:
This is for showing border below input field. 
![image](https://user-images.githubusercontent.com/10433274/130242675-ff8f473b-7c39-4101-8b32-93407c04abfa.png)

Not sure how to use divider prop.

Reply:
There is a `divider` prop you can add to the `pxb-info-list-item` like so
```
<pxb-info-list-item divider="partial">...
```

here's an example from our storybook
https://pxblue-components.github.io/angular/?path=/story/components-info-list-item--within-a-full-list

Reply:
I misunderstood earlier; that divider I was referring to wasn't for the InfoListItem, it was used in the input field's bottom border.